src/System.ValueTuple/tests/ExtensionsTests.cs

```diff
@@ -673,27 +672,27 @@ private struct IntHolder
         public override string ToString()
         {
             var builder = new StringBuilder();
-            if (x1 > 0) { builder.Append($"{x1}"); }
-            if (x2 > 0) { builder.Append($" {x2}"); }
-            if (x3 > 0) { builder.Append($" {x3}"); }
-            if (x4 > 0) { builder.Append($" {x4}"); }
-            if (x5 > 0) { builder.Append($" {x5}"); }
-            if (x6 > 0) { builder.Append($" {x6}"); }
-            if (x7 > 0) { builder.Append($" {x7}"); }
-            if (x8 > 0) { builder.Append($" {x8}"); }
-            if (x9 > 0) { builder.Append($" {x9}"); }
-            if (x10 > 0) { builder.Append($" {x10}"); }
-            if (x11 > 0) { builder.Append($" {x11}"); }
-            if (x12 > 0) { builder.Append($" {x12}"); }
-            if (x13 > 0) { builder.Append($" {x13}"); }
-            if (x14 > 0) { builder.Append($" {x14}"); }
-            if (x15 > 0) { builder.Append($" {x15}"); }
-            if (x16 > 0) { builder.Append($" {x16}"); }
-            if (x17 > 0) { builder.Append($" {x17}"); }
-            if (x18 > 0) { builder.Append($" {x18}"); }
-            if (x19 > 0) { builder.Append($" {x19}"); }
-            if (x20 > 0) { builder.Append($" {x20}"); }
-            if (x21 > 0) { builder.Append($" {x21}"); }
+            if (x1 > 0) { builder.Append(x1); }
+            if (x2 > 0) { builder.Append(" " + x2); }
+            if (x3 > 0) { builder.Append(" " + x3); }
+            if (x4 > 0) { builder.Append(" " + x4); }
+            if (x5 > 0) { builder.Append(" " + x5); }
+            if (x6 > 0) { builder.Append(" " + x6); }
+            if (x7 > 0) { builder.Append(" " + x7); }
+            if (x8 > 0) { builder.Append(" " + x8); }
+            if (x9 > 0) { builder.Append(" " + x9); }
+            if (x10 > 0) { builder.Append(" " + x10); }
+            if (x11 > 0) { builder.Append(" " + x11); }
+            if (x12 > 0) { builder.Append(" " + x12); }
+            if (x13 > 0) { builder.Append(" " + x13); }
+            if (x14 > 0) { builder.Append(" " + x14); }
+            if (x15 > 0) { builder.Append(" " + x15); }
+            if (x16 > 0) { builder.Append(" " + x16); }
+            if (x17 > 0) { builder.Append(" " + x17); }
+            if (x18 > 0) { builder.Append(" " + x18); }
+            if (x19 > 0) { builder.Append(" " + x19); }
+            if (x20 > 0) { builder.Append(" " + x20); }
+            if (x21 > 0) { builder.Append(" " + x21); }
```

Comment:
If this is going to be reverted, the changes in ValueTupleTests.cs seems related and can also be rolled back (except for the file rename/move).